integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -27,19 +28,29 @@ def test_multiple_granules_happy_path(self):
             createdAt_time = int((time.time() + 5) * 1000)
             collection_name = uuid.uuid4().__str__()
             collection_version = uuid.uuid4().__str__()
-            # standard bucket where initial file exists
+            # standard bucket where test files will be copied
             cumulus_bucket_name = "orca-sandbox-s3-provider"
             recovery_bucket_name = helpers.recovery_bucket_name
             excluded_filetype = []
-
-            name_1 = "ancillary_data_input_forcing_ECCO_V4r4.tar.gz"
```

Comment:
replaced .tar.gz with .tif file since uploading the previous 191GB tar file to source bucket takes long.

Reply:
We do need the large file to test timeouts, unfortunately. If you want to be fancy, you could have a `use_large_file = true` flag, but this file might just not be randomizable right now.

Reply:
if we want to keep the large file, does this mean we have to assume the file will already exist in the source bucket so that the long upload is not needed ?
This also means file randomization is not possible right now for large file.

Reply:
I have reverted the changes so that .tar.gz will stay in source bucket and file randomization will not occur due to large upload time.